static/templates/user_profile_modal.handlebars

```diff
@@ -29,7 +29,7 @@
                         <div class="input" contenteditable="false" style="display: none;"></div>
                     </div>
                     {{else}}
-                    <div class="user-profile-modal-field-value">{{this.value}}</div>
+                    <div data-type="{{this.type}}" class="user-profile-modal-field-value">{{this.value}}</div>
```

Comment:
We should store field type in `user-profile-key-value-section` class instead of this, cause that way we can also get user type fields in future, if we want.